ui/components/app/simulation-details/simulation-details.tsx

```diff
@@ -16,8 +20,8 @@ import Preloader from '../../ui/icon/preloader/preloader-icon.component';
 import { BalanceChangeList } from './balance-change-list';
 import { useBalanceChanges } from './useBalanceChanges';
 
-export type SimulationPreviewProps = {
-  simulationData?: SimulationData;
+export type SimulationDetailsProps = {
+  simulationData?: Partial<SimulationData>;
```

Comment:
Why is this `Partial` now, the type should be respected by the backend meaning it should be `undefined` or all present?

Reply:
Done.